doc/source/extending-jupyterhub.rst

```diff
@@ -141,8 +142,14 @@ provide a new cluster size as a command line option ``--size``:
                 --size <NEW-SIZE> \
                 --zone <YOUR-CLUSTER-ZONE>
 
-To display the cluster's name, zone, or current size, use the command
-``gcloud container clusters list``.
+To display the cluster's name, zone, or current size, use the command:
+
+.. code-block:: bash
+
+   gcloud container clusters list
+
+After resizing the cluster, it may take a couple of minutes for the new cluster
+size to be reported back as the service is adding or removing nodes.
```

Comment:
You can find the true count of currently 'ready' nodes by doing:

```
kubectl get node
```

This will report all the nodes and their status (Ready or NotReady)